src/activity/activity.jsx

```diff
@@ -0,0 +1,141 @@
+import { h, Component } from 'preact'
+import cx from 'classnames'
+
+import * as _ from 'utils'
```

Comment:
these are not really like lodash, I used the underscore to have a short namespace ;)